keyboards/annepro2/annepro2.c

```diff
@@ -155,22 +155,22 @@ bool process_record_kb(uint16_t keycode, keyrecord_t *record) {
             case KC_AP2_BT1:
                 annepro2_ble_broadcast(0);
                 /* FIXME: This hardcodes col/row position */
-                ap2_led_blink(0, 1, blue, 8, 50);
+                ap2_led_blink(record->event.key.row, record->event.key.col, blue, 8, 50);
                 return false;
```

Comment:
Note that if someone decides to put `KC_AP2_BT1` on a combo, `record->event.key` will contain `{KEYLOC_COMBO, KEYLOC_COMBO}` (`{254, 254}`) instead of real row and column numbers. If the LED firmware won't handle such out of range values correctly, you should add range checks somewhere.

And the FIXME can probably be removed too.